CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 # This is the legacy minimum version flatbuffers supported for a while.
 cmake_minimum_required(VERSION 3.8...3.25.2)
+cmake_policy(VERSION 3.8)
 
 # Attempt to read the current version of flatbuffers by looking at the latest tag.
-include(CMake/Version.cmake)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/CMake)
```

Comment:
Minor change: the CMake subdirectory is now added to the CMake module path.  This was needed to make `find_package()` work.